<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChoiceWidget</class>
 <widget class="QWidget" name="ChoiceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>531</width>
     <height>61</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Choose the visualization mode&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="audioCircle">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>187</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Audio Circle</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="allThree">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>400</y>
     <width>187</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>All three</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="audioFace">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>187</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Audio Face</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="audioRectangles">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>187</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Audio Rectangles</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>320</y>
     <width>187</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Audio Chart</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
